quic/s2n-quic-qns/etc/Dockerfile.build

```diff
@@ -1,28 +1,15 @@
-FROM rust:latest as planner
-WORKDIR app
-RUN cargo install cargo-chef --version 0.1.23
-COPY Cargo.toml /app
-COPY common /app/common
-COPY quic /app/quic
-COPY tools/xdp /app/tools/xdp
-# Don't include testing crates
-RUN rm -rf quic/s2n-quic-bench quic/s2n-quic-events quic/s2n-quic-sim
-RUN cargo chef prepare  --recipe-path recipe.json
-
-FROM rust:latest as cacher
-WORKDIR app
-RUN cargo install cargo-chef --version 0.1.23
-COPY --from=planner /app/recipe.json recipe.json
-RUN cargo chef cook --recipe-path recipe.json
+FROM rust:latest AS base
 
-FROM rust:latest AS builder
 WORKDIR app
 
 RUN set -eux; \
   apt-get update; \
   apt-get install -y cmake clang;
 
-# copy sources
+RUN cargo install cargo-chef --version 0.1.23
```

Comment:
nit: should we use the latest?

cargo-chef = "0.1.61"

Reply:
I tried using it but it complained about some missing packages in the workspace so I punted on it since this is causing our release CI to fail. I can open an issue to upgrade it later, though.

Reply:
#1845